setup/start.sh

```diff
@@ -82,6 +82,11 @@ if [ ! -f $STORAGE_ROOT/mailinabox.version ]; then
 	chown $STORAGE_USER.$STORAGE_USER $STORAGE_ROOT/mailinabox.version
 fi
 
+# Default policy (initial) for MTA_STS = testing in the current state of inclusion.
+# it can be changed to "none", "testing" or "enforce". With this extention, this is preserved by
+# future upgrades
+
+MTA_STS="${DEFAULT_MTA_STS:-testing}"
```

Comment:
This is cool but I think we may want to just force everyone into testing mode for now, and then later force everyone into enforce when we're confident that MTA-STS is working.

Reply:
Basically everyone is pushed into testing mode, until someone decides to change it. Then, the new setting will be maintained. I think this is a logical approach.

Reply:
i forgot to close this one, it's cool;)